# 1/0: if the model predicts steady box size (1) or not (0)
int32 predict_steady_box_size

# 1/0: if the model predicts steady box position (1) or not (0)
int32 predict_steady_box_position

# 1/0: if the model predicts the box is small (1) or not (0)
int32 predict_small_box

# 1/0: if the model predicts the box is on the left side (1) or not (0)
int32 predict_box_on_left
